spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -348,6 +348,7 @@ object SpdxDeclaredLicenseMapping {
         "Historical Permission Notice and Disclaimer (HPND)" to HPND.toExpression(),
         "IBM Public License" to IPL_1_0.toExpression(),
         "icu-unicode license" to ICU.toExpression(),
+        "Indiana University Extreme! Lab Software License, vesion 1.1.1" to licenseRef("scancode", "indiana-extreme"),
```

Comment:
Maybe the commit message could emphasize that the typo in "vesion" is indeed intended.

Reply:
Done.